app/presenters/badge_claim_presenter.rb

```diff
@@ -0,0 +1,16 @@
+class BadgeClaimPresenter
+  attr_reader :claim
+
+  def initialize(claim)
+    @claim = claim
+  end
+
+  def as_json(opt = {})
+    {
+        id: claim.id,
```

Comment:
In the migration, you add timestamps to this (updated_at and created_at), it may make sense to display them in the JSON.